index.js

```diff
@@ -1056,6 +1063,42 @@
     return ap(ap(map(f, x), y), z);
   };
 
+  //# apFirst :: Apply f => (f a, f b) -> f a
+  //.
+  //. Combine two effectful actions, keeping only the result of the first.
+  //. Equivalent to Haskell's (<*) function.
```

Comment:
Let's wrap the function name in backticks so it will be rendered as "`(<*)`" rather than "(<*)".